CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+# Unreleased
+
+N/A
+
+# Release 1.0.0
+* Initial Release
+* Library now uses semantic versioning
+* Updated documentationt
+* Increased so.version to 3.0
```

Comment:
```suggestion
* Increased soversion to 3.0
```
The dot looks weird to me, suggest removing it.

Reply:
done